generics/services/survey.js

```
@@ -383,11 +383,116 @@ const deleteSolutionResource = function (solutionIds, resourceType, tenantId, or
+						result = response.result
+					} else {
+						result.success = false
+					}
+				}
+
+				return resolve(result)
+			}
+		} catch (error) {
+			return reject(error)
+		}
+	})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)